NitroxPatcher/Patches/Dynamic/CyclopsDestructionEvent_OnConsoleCommand_Patch.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Reflection;
+using HarmonyLib;
+using NitroxClient.Communication.Abstract;
+using NitroxClient.MonoBehaviours;
+using NitroxModel.DataStructures;
+using NitroxModel.DataStructures.Util;
+using NitroxModel.Helper;
+using NitroxModel.Packets;
+using NitroxModel_Subnautica.DataStructures;
+using NitroxModel_Subnautica.Packets;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+public class CyclopsDestructionEvent_OnConsoleCommand_Patch : NitroxPatch, IDynamicPatch
+{
+    private static readonly MethodInfo TARGET_METHOD_RESTORE = Reflect.Method((CyclopsDestructionEvent t) => t.OnConsoleCommand_restorecyclops(default));
+    private static readonly MethodInfo TARGET_METHOD_DESTROY = Reflect.Method((CyclopsDestructionEvent t) => t.OnConsoleCommand_destroycyclops(default));
+
+    public static bool PrefixRestore()
+    {
+        ErrorMessage.AddWarning("Cyclops restoring is not available in Nitrox");
```

Comment:
There is already a translated error message for those cases, you can have a look at speed, sub console commands patches